packages/forms/frontend/sirius-components-forms/src/propertysections/SplitButtonPropertySection.tsx

```diff
@@ -243,7 +245,7 @@ export const SplitButtonPropertySection = ({
   };
 
   return (
-    <div>
+    <div style={{ display: 'flex', flexDirection: 'row' }}>
```

Comment:
Why did you use an inline style here instead of creating a new style rule like you did in `ButtonPropertySection.tsx` for example?

Reply:
Here the class variable is an array because it creates a class for each button of the split button.
In my case I want to apply a style the container div so I have only one instance.
If you have a better implementation I gladly change it.

Reply:
Ok I see your problem.
You could have created a second makeStyle variable with only one style containing `display: 'flex', flexDirection: 'row',`.

Reply:
Yes I will do it mainly due to comment of Stephanne on the margin/gap of the PropertyLabelSection

Reply:
Done